nautobot/core/views/__init__.py

```diff
@@ -177,28 +178,31 @@ def get(self, request):
-                view_func = resolve(reverse(url)).func
-                # For a UIViewSet, view_func.cls gets what we need; for an ObjectListView, view_func.view_class is it.
-                view_or_viewset = getattr(view_func, "cls", getattr(view_func, "view_class", None))
-                queryset = view_or_viewset.queryset.restrict(request.user, "view")
-                # For a UIViewSet, .filterset_class, for an ObjectListView, .filterset.
-                filterset = getattr(view_or_viewset, "filterset_class", getattr(view_or_viewset, "filterset", None))
-                # For a UIViewSet, .table_class, for an ObjectListView, .table.
-                table = getattr(view_or_viewset, "table_class", getattr(view_or_viewset, "table", None))
-
-                # Construct the results table for this object type
-                filtered_queryset = filterset({"q": form.cleaned_data["q"]}, queryset=queryset).qs
-                table = table(filtered_queryset, orderable=False)
-                table.paginate(per_page=SEARCH_MAX_RESULTS)
-
-                if table.page:
-                    results.append(
-                        {
-                            "name": queryset.model._meta.verbose_name_plural,
-                            "table": table,
-                            "url": f"{reverse(url)}?q={form.cleaned_data.get('q')}",
-                        }
-                    )
+                try:
+                    view_func = resolve(reverse(url)).func
+                    # For UIViewSet, view_func.cls gets what we need; for an ObjectListView, view_func.view_class is it.
+                    view_or_viewset = getattr(view_func, "cls", getattr(view_func, "view_class", None))
+                    queryset = view_or_viewset.queryset.restrict(request.user, "view")
+                    # For a UIViewSet, .filterset_class, for an ObjectListView, .filterset.
+                    filterset = getattr(view_or_viewset, "filterset_class", getattr(view_or_viewset, "filterset", None))
+                    # For a UIViewSet, .table_class, for an ObjectListView, .table.
+                    table = getattr(view_or_viewset, "table_class", getattr(view_or_viewset, "table", None))
+
+                    # Construct the results table for this object type
+                    filtered_queryset = filterset({"q": form.cleaned_data["q"]}, queryset=queryset).qs
+                    table = table(filtered_queryset, orderable=False)
+                    table.paginate(per_page=SEARCH_MAX_RESULTS)
+
+                    if table.page:
+                        results.append(
+                            {
+                                "name": queryset.model._meta.verbose_name_plural,
+                                "table": table,
+                                "url": f"{reverse(url)}?q={form.cleaned_data.get('q')}",
+                            }
+                        )
+                except NoReverseMatch:
```

Comment:
As we have `staticgroup` registered as a searchable model but don't yet have a UI view for the model, this exception is thrown. Handling it gracefully here makes the test `nautobot.core.tests.test_views.HomeViewTestCase.test_search` pass instead of failing for now, but we should probably enhance the test so that it will fail if any "Missing URL" messages are shown in the response?